.github/workflows/build-llvm.yml

```diff
@@ -6,6 +6,27 @@ on:
 name: Build LLVM Binaries
 
 jobs:
+  rv32e-linux-x86-64:
+    name: rv32e Linux x86-64
+    runs-on: ubuntu-20.04
+    steps:
```

Comment:
what about the all the other platforms that solang supports?

Reply:
Also you've not added it to the ci image

Reply:
> what about the all the other platforms that solang supports?

For now we only need Linux.. maybe ARM MacOS